src/field_theory/ax_grothendieck.lean

```diff
@@ -0,0 +1,108 @@
+/-
+Copyright (c) 2023 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes
+-/
+import data.mv_polynomial.basic
+import ring_theory.algebraic
+import data.fintype.card
+import data.zmod.basic
+
+/-!
+# Ax-Grothendieck for algebraic extensions of `zmod p`
+
+This file proves that if `K` is an algebraic extension of `zmod p` for some prime `p`,
+then any injective polynomial map `K^n -> K^n` is surjective.
```

Comment:
I'm a bit confused by this explanation; `zmod` and `prime` are not mentioned anywhere in the rest of the file.

Reply:
Should it say "some finite field (such as `zmod p` for some prime `p`)" or similar?

Reply:
A field is an algebraic extension of a finite field iff it is an algebraic extension of `zmod p` for some `p` (every finite field is an algebraic extension of `zmod p` for some `p`, and algebraic extensions of algebraic extensions are also algebraic). So this is basically the same as the statement in the file, and arguably a more natural way to state the theorem in English.

Reply:
I think it is best to change it though actually.

Reply:
fixed